requirements.txt

```diff
@@ -90,7 +90,7 @@ python3-openid==3.1.0 ; python_version >= '3.0'
 pytz==2018.9
 raven==6.9.0
 razorpay==1.2.0
-redis==2.10.6
+redis==3.2.0
```

Comment:
One more thing is that I noticed that botocore and some other packages are also upgraded in Pipfile.lock, could you please update the requirements files by ```pipenv lock -r > requirements.txt``` and ```pipenv lock -r -d > requirements_dev.txt``` as well?